fl4health/utils/nnunet_utils.py

```diff
@@ -423,6 +423,7 @@ def __init__(
         self._step_count: int
         super().__init__(optimizer, -1, False)
 
+    # mypy incorrectly infers get_lr returns a float
```

Comment:
Is this a bug that we can link too? If it's more subtle than that, maybe we can briefly describe why it happens?